packages/react/use-body-pointer-events/src/useBodyPointerEvents.tsx

```diff
@@ -33,6 +33,7 @@ function useBodyPointerEvents({ disabled }: { disabled: boolean }) {
       }
 
       function resetPointerEvents() {
+        changeCount--;
```

Comment:
I think that's my bad on this one, when we rejigged the code before I went on holidays I didn't catch that this bit wasn't moved with it.

Reply:
Nah, my fault for not moving it 😅  complete oversight.